skin.css

```diff
@@ -10,4 +10,41 @@ button.btn {
   line-height: 1.15;
   text-align: center;
   vertical-align: middle;
+}
+.btn{
+  background-color: white; 
+  color: rgba(78, 3, 177, 0.945);
+  padding: 14px 58px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin-left: 14px;
+  
+}
+button{
+  -webkit-transition-duration: 0.4s; /* Safari */
+  transition-duration: 0.4s;
+}
+
+button:hover{
+  background-color: white;
+  color: rgba(111, 0, 255, 0.589);
+
+}
+.btn--primary{
+  background-color: rgba(58, 3, 161, 0.815);
+  color: white;
+  padding: 17px 59px;
+  margin-left: 10px;

```

Comment:
Not sure why you're overriding the padding & margin here - should maybe be in the `.btn` ruleset